tendrl/notifier/objects/definition/notifier.yaml

```diff
@@ -163,51 +172,4 @@ namespace.alerting:
       value: alerting/notification_medium/supported/
       help: "Supported means of notification"
       enabled: true
-    AlertTypes:
-      attrs:
-        type: List
-        help: 'A Dict of integration to types of alerts the integration handles'
-      list: alerting/alert_types
-      enabled: true
-      value: alerting/alert_types
-      help: 'Alert types'
-    NotificationConfig:
-      enabled: True
-      attrs:
-        config:
-          help: "Notification global configuration"
-          type: Dict
-      value: notification_settings/
-      list: notification_settings/
-      help: "Notification configuration"
-    ClusterAlertCounters:
-      enabled: True
-      attrs:
-        warning_count:
-          help: "Count of warning alerts for the cluster"
-          type: int
-        critical_count:
-          help: "Count of critical alerts for the cluster"
-          type: int
-        integration_id:
-          help: "Id of the cluster"
-          type: String
-      value: /clusters/{0}/alert_counters
-      list: /clusters/{0}/alert_counters
-      help: "Cluster Alert Counter"
-    NodeAlertCounters:
```

Comment:
Yes this is required as per https://github.com/Tendrl/gluster-integration/issues/356. We should have alerts counts maintained at cluster, volume and node level